sys-kernel/coreos-sources/coreos-sources-3.18.0.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+ETYPE=sources
+K_SECURITY_UNSUPPORTED=1
+inherit kernel-2
+detect_version
+detect_arch
+
+inherit git-2 versionator
```

Comment:
What's versionator for?

Also cloning the whole kernel git tree tends to be kinda slow, it may be better to fetch a snapshot tarball similar to: https://github.com/coreos/coreos-overlay/blob/master/sys-apps/efunctions/efunctions-1.ebuild